personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -12,16 +12,53 @@ function PersonalityTestXBlockStudent(runtime, element) {
         return tmp;
     }
 
-    function paragraphize(str) {
+    function paragraphize (str) {
         var tmp = str.split('\n');
-        var buff = document.createElement('div');
+        var buff = $('<div />');
         tmp.forEach(function (item) {
-            var p = document.createElement('p');
-            p.appendChild(document.createTextNode(item));
-            buff.appendChild(p);
+            var p = $('<p />');
+            p.text(item);
+            buff.append(p);
         });
         return buff;
     }
+    function addQuestion (question, list, listCounter, studentAnswers) {
```

Comment:
We had a look with Jeremy on the css but it was destroying the display. So I did it like that